pkg/controller/etcdproxy/etcdproxy_controller.go

```diff
@@ -33,31 +33,41 @@ import (
 const httpUserAgentName = "etcdproxy-controller"
 
 const (
-	// SuccessSynced is used as part of the Event 'reason' when an EtcdStorage is synced.
-	SuccessSynced = "Synced"
-	// Terminating is used as part of the Event 'reason' when an EtcdStorage resource is being deleted (deletion
-	// timestamp is set).
-	Terminating = "Terminating"
-	// ResourceReclaimed is used as part of the Event 'reason' when a replicaset or service already exists
-	// and EtcdStorage reclaims it.
+	// EtcdStorageSynced is used as part of the Event reason when an EtcdStorage resource is successfully synced.
+	EtcdStorageSynced = "EtcdStorageSynced"
+	// MessageResourceSynced is the message used for an Event when an EtcdStorage resource is synced successfully.
+	MessageEtcdStorageSynced = "EtcdStorage %q synced successfully"
```

Comment:
I am not a big friend of centralized strings. It's much harder to understand when something is wrong. Also golint cannot help with wrong string patterns.